examples/aws/ingress/values.yaml

```diff
@@ -1,14 +1,21 @@
 # This example shows how to deploy and expose GraphDB with AWS Load Balancer Controller Ingress without SSL.
 
 configuration:
-  externalUrl: http://graphdb-example-dns-name.com/ # Change this to your ALB DNS name or Route53 if you use it.
+  # Change this to your AzureDNS domain name or use nip.io which you can do after you deploy the chart since you
+  # need to map the public IP address of the Application Gateway to the nip.io.
+  externalUrl: http://ip.address.nip.io/
 
 ingress:
   enabled: true
   className: alb
   annotations:
+   # Define name of the Load Ingress
```

Comment:
These comments are superfluous IMO, use comments to describe the _why_, not the _what_. Or when something is truly opaque.